rero_mef/api_mef.py

```diff
@@ -40,6 +42,42 @@ class EntityMefRecord(ReroMefRecord):
     search = None
     mef_type = ''
 
+    def set_deleted(self):
+        """Set deleted.
+
+        Sets MEF deleted value from sources.
+        """
+        changed = False
+        source_data = deepcopy(self).replace_refs()
+        if sources := source_data['sources']:
+            for source in sources:
+                if deleted := self[source].get('deleted'):
+                    self['deleted'] = deleted
+                    changed = True
+                    break
```

Comment:
Is it possible that more than one source has been deleted?

Reply:
Yes, but here we need to know only if one has be deleted.